filesystems/s3/src/main/java/org/lerch/s3fs/S3FileSystemProvider.java

```diff
@@ -69,17 +66,18 @@
  * </p>
  */
 public class S3FileSystemProvider extends FileSystemProvider {
-
-    private static final String AMAZON_S3_FACTORY_CLASS = "s3fs_amazon_s3_factory";
+    public static final String AMAZON_S3_FACTORY_CLASS = "s3fs_amazon_s3_factory";
```

Comment:
Is this constant used somewhere outside of this class?

Reply:
No. I will keep it private until it is.